utils/utils.go

```diff
@@ -79,10 +81,14 @@ func LoadSession(mm *pluginapi.Client, sessionID, actingUserID string) (*model.S
 	return session, nil
 }
 
-func ClientFromSession(mm *pluginapi.Client, mattermostSiteURL, sessionID, actingUserID string) (*model.Client4, error) {
+func ClientFromSession(mm *pluginapi.Client, mattermostSiteURL, sessionID, actingUserID string, i18nBundle *i18n.Bundle) (*model.Client4, error) {
+	loc := i18nBundle.GetUserLocalizer(actingUserID)
 	session, err := LoadSession(mm, sessionID, actingUserID)
 	if err != nil {
-		return nil, errors.Wrap(err, "failed to load session")
+		return nil, errors.Wrap(err, i18nBundle.LocalizeDefaultMessage(loc, &nsi18n.Message{
+			ID:    "apps.command.list.table.header",
+			Other: "failed to load session",
+		}))
```

Comment:
This message id doesn't seem right. I don't see `failed to load session` in the i18n json file either